packages/app/src/app/components/WorkspaceSetup/steps/Create.tsx

```diff
@@ -70,6 +70,7 @@ export const Create: React.FC<StepProps> = ({
 
         setQueryParam('workspace', team.id);
         setFreshWorkspaceId(team.id);
+        await actions.dashboard.getTeams();
```

Comment:
Does this fix the issue with the sidebar, which couldn't find the current workspace?

Reply:
Yes, I think so. Weirdly on stream that problem was not reproducing, I'm curios what happens when this goes to prod